the-toad-tribune/src/layout/Navgation.tsx

```diff
@@ -1,39 +1,67 @@
-import React, { KeyboardEvent} from "react";
+import React, { KeyboardEvent, useState } from "react";
 import styled from "styled-components";
 import { getNewsEverything } from "../api/newsApi";
 import { NewsEverythingRequest } from "../api";
 
-
 interface INavProps {
   renderMainLayoutPage: Function;
   renderSearchPage: Function;
-  searchValue: string;
   setSearchValue: Function;
   setSearchResults: Function;
 }
 
-const Navigation: React.FC<INavProps> = ({ renderMainLayoutPage, renderSearchPage, searchValue, setSearchResults, setSearchValue }) => {
+const Navigation: React.FC<INavProps> = ({
+  renderMainLayoutPage,
+  renderSearchPage,
+  setSearchResults,
+  setSearchValue,
+}) => {
+  const [validateSearchValue, setValidateSearchValue] = useState("");
+  const [searchError, setSearchError] = useState("");
```

Comment:
It would be good to annotate this as `useState<string>("");` so that we'll get an error if we try to set it to anything other than a string.

Reply:
Thankie. Fixed it